### ISSUE TITLE
TypeError: Object doesn't support property or method 'find'

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#321]
https://sentry.io/5calls/5-calls/issues/290002730/

```
TypeError: Object doesn't support property or method 'find'
  at Anonymous function (/js/app.js:6:26322)
  at reducers.mergeIssues (/js/app.js:6:26259)
  at Anonymous function (/js/app.js:1:7581)
  at Anonymous function (/js/app.js:4:11767)
  at Anonymous function (/js/app.js:4:11492)
```

**Comment by nickoneill** [comment]
Still figuring out how I can make these reports public...

A couple reports of this for windows users since the category stuff went out last night. This one is in the `mergeIssues` function.

**Comment by nickoneill** [comment]
Pushed a fix with this: d46a6650d46300cbe076aff155f8dced952523ad

**Comment by nickoneill** [comment]
No more cases of this overnight. Closing.
